app/javascript/src/components/Dashboard/Notes/Note.jsx

```diff
@@ -0,0 +1,51 @@
+import React from "react";
+
+import { Clock, MenuVertical } from "neetoicons";
+import { Avatar, Dropdown, Tag, Tooltip, Typography } from "neetoui";
+
+import { createdAgoFormat, createdAtFormat } from "utils/dateFormats";
+
+const Note = ({ note }) => (
+  <div className="mb-3 w-full  border border-gray-300 bg-white p-4 shadow-md  dark:border-gray-700 dark:bg-gray-800 ">
+    <div className="flex justify-between">
+      <Typography style="h4" weight="bold">
+        {note.title}
+      </Typography>
+      <Dropdown buttonStyle="text" icon={MenuVertical}>
+        <li>Edit</li>
+        <li>Delete</li>
+      </Dropdown>
+    </div>
+    <div className="mb-2">
+      <Typography className="--neeto-ui-gray-600">
+        {note.description}
+      </Typography>
+    </div>
```

Comment:
Why do we need this div here? What's the issue with keeping this classname within `Typography`?